.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=vulnerability
```

Comment:
vulnerability?

Reply:
For development, we need to connect to local VMaaS. We can only do this by running in a single project.